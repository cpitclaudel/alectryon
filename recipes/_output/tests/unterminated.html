<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" class="alectryon-standalone" xml:lang="en" lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="Docutils 0.19: https://docutils.sourceforge.io/" />
<title>Parsing errors in reST</title>
<link rel="stylesheet" href="alectryon.css" type="text/css" />
<link rel="stylesheet" href="docutils_basic.css" type="text/css" />
<link rel="stylesheet" href="pygments.css" type="text/css" />
<script type="text/javascript" src="alectryon.js"></script>
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/IBM-type/0.5.4/css/ibm-type.min.css" integrity="sha512-sky5cf9Ts6FY1kstGOBHSybfKqdHR41M0Ldb0BjNiv3ifltoQIsg0zIaQ+wwdwgQ0w9vKFW7Js50lxH9vqNSSw==" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/firacode/5.2.0/fira_code.min.css" integrity="sha512-MbysAYimH1hH2xYzkkMHB6MqxBqfP0megxsCLknbYqHVwXTCg9IqHbk+ZP/vnhO8UEW6PaXAkKe2vQ+SWACxxA==" crossorigin="anonymous" />
<meta name="viewport" content="width=device-width, initial-scale=1">
</head>
<body>
<div class="alectryon-root alectryon-centered"><div class="alectryon-banner">Built with <a href="https://github.com/cpitclaudel/alectryon/">Alectryon</a>, running Coq+SerAPI-noexec. Bubbles (<span class="alectryon-bubble"></span>) indicate interactive fragments: hover for details, tap to reveal contents. Use <kbd>Ctrl+↑</kbd> <kbd>Ctrl+↓</kbd> to navigate, <kbd>Ctrl+🖱️</kbd> to focus. On Mac, use <kbd>⌘</kbd> instead of <kbd>Ctrl</kbd>.</div><div class="document" id="parsing-errors-in-rest">
<h1 class="title">Parsing errors in reST</h1>

<p>To run:</p>
<pre class="literal-block">
alectryon unterminated.rst --coq-driver sertop_noexec &gt; unterminated.rst.out 2&gt;&amp;1; \
  echo &quot;exit: $?&quot; &gt;&gt; unterminated.rst.out
    # Coq → reST; produces ‘unterminated.rst.out’
</pre>
<pre class="alectryon-io highlight"><!-- Generator: Alectryon --><span class="alectryon-txt"><span class="c">(* &quot;This string is unterminated. *)</span></span></pre><div class="system-messages section">
<h1>Docutils System Messages</h1>
<div class="system-message">
<p class="system-message-title">System Message: ERROR/3 (<tt class="docutils">tests/unterminated.rst</tt>, line 13)</p>
Coq raised an exception:
  &gt; Syntax Error: Lexer: Unterminated string
The offending chunk is delimited by &gt;&gt;&gt;…&lt;&lt;&lt; below:
  &gt; (* &gt;&gt;&gt;&quot;This string is unterminated. *)&lt;&lt;&lt;
Results past this point may be unreliable.</div>
</div>
</div>
</div></body>
</html>
