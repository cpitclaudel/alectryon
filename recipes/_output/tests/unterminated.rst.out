tests/unterminated.rst:13: (ERROR/3) Coq raised an exception:
  > Syntax Error: Lexer: Unterminated string
The offending chunk is delimited by >>>â€¦<<< below:
  > (* >>>"This string is unterminated. *)<<<
Results past this point may be unreliable.
tests/unterminated.rst:13: (WARNING/2) Orphaned message for sid b'0':
 >  Not interpreting "*)" as the end of current
 >  non-terminated comment because it occurs in a
 >  non-terminated string of the comment.
 >  [comment-terminator-in-string,parsing,default]
exit: 13
